PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/CONTRIBUTING.MD

```diff
@@ -0,0 +1,15 @@
+### Configuring MS Application Insights
+- An Azure account with an active subscription.
+
+#### 1. Generate the Instrumentation Key
+
```

Comment:
If you're going to have a heading for "Generate the Instrumentation Key" you should have some information under that heading. Even if you just provide a link to external documentation and say: "You can learn about generating the instrumentation key from [link here]", it would still be better than leaving an empty heading.